src/app/context/wallet/WalletConnectionProvider.tsx

```diff
@@ -1,119 +1,51 @@
-import {
-  TomoContextProvider,
-  useTomoModalControl,
-  useTomoProviders,
-  useTomoWalletConnect,
-  useTomoWalletState,
-} from "@tomo-inc/wallet-connect-sdk";
-import "@tomo-inc/wallet-connect-sdk/style.css";
-import { useTheme } from "next-themes";
-import {
-  createContext,
-  useCallback,
-  useContext,
-  type PropsWithChildren,
-} from "react";
+"use client";
 
-import { getNetworkConfig } from "@/config/network.config";
-import { keplrRegistry } from "@/config/wallet/babylon";
+import { Network, WalletProvider } from "@babylonlabs-io/bbn-wallet-connect";
+import { type PropsWithChildren } from "react";
 
-// We have to split the connection into two parts
-// so we can use the tomoWalletConnect and tomoModal hooks
-export const WalletConnectionProvider = ({ children }: PropsWithChildren) => {
-  const { resolvedTheme } = useTheme();
+import { ErrorState } from "@/app/types/errors";
+import { keplrRegistry } from "@/config/wallet/babylon";
```

Comment:
I suggest we stop using `keplrRegistry` which was a part of Tomo wallet connect and use `bbnDevnet` instead. We will also have `bbnTestnet` later on or so, so these files describe chains, `ChainInfo` in Keplr